test/bigtest/tests/resource-managed-coverage-test.js

```diff
@@ -25,7 +26,7 @@ describeApplication('ResourceManagedCoverage', () => {
 
   describe('visiting the resource page with managed coverage undefined', () => {
     beforeEach(function () {
-      return this.visit(`/eholdings/resources/${resource.id}`, () => {
+      this.visit(`/eholdings/resources/${resource.id}`, () => {
```

Comment:
Forgot to remove the `visit` callback for this file

Reply:
Missed all of them in this file. Fixed.